modules/candidate_parameters/jsx/familyInfo.js

```diff
@@ -1,353 +1,374 @@
 /* exported RFamilyInfo */
 
-var FamilyInfo = React.createClass(
-  {
-    getInitialState: function() {
-      return {
-        relationshipOptions: {
-          "full_sibling": "Full Sibling",
-          "half_sibling": "Half Sibling",
-          "1st_cousin": "First Cousin"
-        },
-        Data: [],
-        formData: {},
-        updateResult: null,
-        errorMessage: null,
-        isLoaded: false,
-        loadedData: 0
-      };
-    },
-    componentDidMount: function() {
-      var that = this;
-      $.ajax(
-                this.props.dataURL,
-        {
-          dataType: 'json',
-          xhr: function() {
-            var xhr = new window.XMLHttpRequest();
-            xhr.addEventListener(
-                            "progress",
-                            function(evt) {
-                              that.setState(
-                                {
-                                  loadedData: evt.loaded
-                                }
-                                );
-                            }
-                        );
-            return xhr;
-          },
-          success: function(data) {
-            that.setState(
-              {
-                Data: data,
-                isLoaded: true
-              }
-                        );
-          },
-          error: function(data, errorCode, errorMsg) {
-            that.setState(
-              {
-                error: 'An error occurred when loading the form!'
+var FamilyInfo = React.createClass({
+  getInitialState: function() {
+    return {
+      relationshipOptions: {
```

Comment:
This doesn't need to be in the state, just declare as a variable inside render method